samples/SPLToJsonConvert/info.xml

```diff
@@ -5,12 +5,12 @@
    <info:name>com.ibm.streamsx.json.SPLToJsonConvert</info:name>
    <info:description></info:description>
    <info:version>1.0.0</info:version>
-   <info:requiredProductVersion>4.0.1.0</info:requiredProductVersion>
+   <info:requiredProductVersion>4.2.0.0</info:requiredProductVersion>
```

Comment:
The sample history shows it was just added to go with some new functions that were not present in older releases, which is why I changed the required product number.